### PR TITLE
documentation(ui5-notification-list): update the documentation

### PR DESCRIPTION
**Pull request by TeodorTaushanov**

### CODE REVIEW

File: packages/fiori/src/NotificationList.ts [hunk 1]
```diff
@@ -36,6 +36,23 @@ type NotificationItemCloseEventDetail = NotificationItemEventDetail;
  * The `ui5-notification-list` web component represents
  * a container for `ui5-li-notification-group` and `ui5-li-notification`.
  *
+ * ### Keyboard Handling
```
Comment (LidiyaGeorgieva) [inline packages/fiori/src/NotificationList.ts:39]: Notification List component doesn't have a page in the website and this information is not visible in the documentation.

File: packages/website/docs/_components_pages/fiori/NotificationList/NotificationList.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Basic from "../../../_samples/fiori/NotificationList/Basic/Basic.md";
+import GroupItems from "../../../_samples/fiori/NotificationList/GroupItems/Basic.md";
+import InShellBar from "../../../_samples/fiori/NotificationList/InShellBar/InShellBar.md";
+
+<%COMPONENT_OVERVIEW%>
+
+## Basic Sample
+<Basic />
+
+## Notification List with Group Items
```
Comment (LidiyaGeorgieva) [inline packages/website/docs/_components_pages/fiori/NotificationList/NotificationList.mdx:10]: I think this should be in "More samples" section, on the bottom of the page.
Only Basic sample to be on the top (before the API description).

File: packages/fiori/src/NotificationList.ts [hunk 3]
```diff
@@ -39,9 +39,26 @@ type NotificationItemCloseEventDetail = NotificationItemEventDetail;
  * The `ui5-notification-list` web component represents
```
Comment (BorisDafov) [inline packages/fiori/src/NotificationList.ts:39]: The recommended structure of a component (see [here](https://workzone.one.int.sap/site#workzone-viewAll?sap-app-origin-hint=&/wiki/show/gnY0lJJRpoHghUaznFjmYo)) also has other sections like Properties, Events, Methods, etc... Do we need these?

File: packages/website/docs/_components_pages/fiori/NotificationList/NotificationList.mdx (hunk 2 above)
Comment (TeodorTaushanov) [reply packages/website/docs/_components_pages/fiori/NotificationList/NotificationList.mdx]: Done.

File: packages/fiori/src/NotificationList.ts (hunk 3 above)
Comment (TeodorTaushanov) [reply packages/fiori/src/NotificationList.ts:39]: They will be generated from the metadata.

**Review by LidiyaGeorgieva — APPROVED** [review]

**Pull request merged**